src/analysis/retail/priest/discipline/modules/features/AtonementHealingBreakdown.tsx

```diff
@@ -53,7 +67,7 @@ const AtonementHealingBreakdown = ({
                 <tr key={ability.guid}>
                   <td style={{ width: '30%' }}>
                     <SpellLink spell={abilityToSpell(ability)} icon={false}>
-                      <Icon icon={ability.abilityIcon} /> {ability.name}
+                      <Icon icon={ability.abilityIcon} /> {getAbilityNameReporting(ability)}
```

Comment:
whats the story here? and does this become simpler by using `<SpellLink spell={ability.guid} />` instead to have the spell data from our internal list looked up?

Reply:
Unsure, I just adapted what was there. Changed it to this.